webrtc.html

```diff
@@ -11209,6 +11205,47 @@ <h2>
                     </p>
                   </li>
                 </ol>
+                <p>
+                  The <dfn class="export">codec match</dfn> algorithm given two {{RTCRtpCodec}} or
+                  compatible dictionaries <var>first</var> and <var>second</var> is as follows:
```

Comment:
Remove "or compatible dictionaries"
```suggestion
                  The <dfn class="export">codec match</dfn> algorithm given two {{RTCRtpCodec}}
                  <var>first</var> and <var>second</var> is as follows:
```